caffe2/serialize/versions.h

```diff
@@ -67,7 +67,11 @@ constexpr uint64_t kProducedFileFormatVersion = 0x3L;
 //  0x3L: (Comment missing)
 //  0x4L: (Comment missing)
 //  0x4L: (update) Added schema to function tuple. Forward-compatible change.
-constexpr uint64_t kProducedBytecodeVersion = 0x4L;
+//  0x5L: (update) Update tensor storage schema. For bytecode archive, the
+//  root key of tensor storage is updated from {index} to {the_pointer_value_the_tensor.storage}.
```

Comment:
Add the comment that bytecode is sharing constant tensor files from torchscript, and only serialize extra tensors that are not in the torchscript constant table.

Reply:
Add it in last commit